external/anomalib/anomaly_classification/openvino.py

```diff
@@ -22,6 +22,7 @@
 from typing import Optional, Union
 
 from addict import Dict as ADDict
+from anomalib.core.model.inference import OpenVinoInferencer
```

Comment:
Should we rename this in `anomalib` to match the new naming convention? Maybe  I could quickly add this to my existing config [MR](https://gitlab-icv.inn.intel.com/algo_rnd_team/anomaly/-/merge_requests/143)

Reply:
That would be nice. Plus I guess this convention might also be something to do with the trademark.

Reply:
in terms of trademark, I agree. It is just against our camel case naming convention, but should be alright

Reply:
Yeah I agree. It breaks our coding convention.